crates/red_knot/src/types.rs

```diff
@@ -529,6 +594,10 @@ impl std::fmt::Display for DisplayType<'_> {
             Type::Never => f.write_str("Never"),
             Type::Unknown => f.write_str("Unknown"),
             Type::Unbound => f.write_str("Unbound"),
+            Type::Module(module_id) => {
+                // NOTE: something like this?: "<module 'module-name' from 'path-from-fileid'>"
+                todo!("{module_id:?}")
```

Comment:
I left this as a TODO for now, but could implement it if you tell me what you'd like. It's the *type* of a module, which doesn't have much info when python prints it out. Probably a bikeshed.
```python
>>> import x
>>> x
<module 'x' from '/home/asteroid/code/redknot.attrchain/x.py'>
>>> type(x)
<class 'module'>
```

Reply:
I think copying the runtime output is reasonable here; totally fine with you either doing it in this PR or leaving it as a todo for later, up to you.

Reply:
Well, `DisplayType` only has access to `TypeStore`. We'd need a `SemanticDb` to call `ModuleTypeId::name` and/or a `Files` to call `Files::path` on `FileId`. I don't think I can access either of these in this context. I'll leave it for now.